massa-client/src/tests/tools.rs

```diff
@@ -4,15 +4,12 @@ use std::fs;
 use time::UTime;
 use toml_edit::{value, Document};
 
-// TODO: move me in massa-node tests to fix #361
-fn _update_genesis_timestamp() {
-    let toml = fs::read_to_string("base_config/config.toml").expect("Unable to read file");
+pub fn update_genesis_timestamp(config_path: &str) {
+    let toml = fs::read_to_string(config_path).expect("Unable to read file");
     let mut doc = toml.parse::<Document>().unwrap();
-    doc["consensus"]["genesis_timestamp"] =
-        value(format!("{}", UTime::now(1000 * 60).unwrap().to_millis()));
-    doc["consensus"]["end_timestamp"] = value(format!(
+    doc["consensus"]["genesis_timestamp"] = value(format!(
         "{}",
-        UTime::now(1000 * 60 * 60).unwrap().to_millis()
+        UTime::now(10000 * 60 * 60).unwrap().to_millis()
```

Comment:
That's not what compensation millis are for, but it works :)

It kinda hides the fact that you are adding 10 000 *60 * 60 to now

Reply:
What can be done to improve the situation?

Reply:
The situation, for the moment we have this `now` function that take a compensation. May be we would like a `now` function (without argument), `now_with_compensation` (with a compensation) and a `now_with_offset` that is basically the same of the previous one but with a more adapted name in this case?

If you want I can create a _good first issue_, I would prefer to do this in an other PR later. :pray:

Reply:
Follow up in #1907